### ISSUE TITLE
Decrease Plugin size

### ISSUE THREAD

**Issue by 123dma** [kmcgill88/admob_flutter#230]
Hi, congratulations for the plugin I was looking for a plugin that would take up less space in the final APK. I have already followed all the procedures to decrease the final APK size suggested by the Flutter documentation.

Currently, the size occupied by this plug-in is similar to firebase_admob 0.10.0 + 2.

Here are some examples.
- APK with plugin firebase_admob 0.10.0 + 2
![firebase_admob 0 10 0+2](https://user-images.githubusercontent.com/8977954/97114103-482dcd00-16cd-11eb-8e78-9012efd7e182.PNG)
- APK with plugin admob_flutter 1.0.0
![admob_flutter 1 0 0](https://user-images.githubusercontent.com/8977954/97114106-4c59ea80-16cd-11eb-811d-9f5b179f4ca0.PNG)
- APK without the 2 plugins
![Sem Admob](https://user-images.githubusercontent.com/8977954/97114108-4f54db00-16cd-11eb-8c46-911fbfe393dd.PNG)

The difference between APK´s is giving 1.3Mb.

I would like to ask if there is any way to decrease the size of this plugin in the final APK. Currently I only use banners and interstitials. I do not use award-winning videos or native ads.  appreciate

**Comment by EliasDeuss** [comment]
@123dma I don't think so.. admob_flutter depends on the amob sdk (same with firebase_admob), 194kb is not a lot of data these days anyway.

**Comment by 123dma** [comment]
Hi @EliasDeuss  thanks for the reply. 

It wouldn't be 194kb, but 1.3Mb that the library increases in the APK.

I believe that a library smaller than admob firebase would be a good upgrade.

Have you tried to use sdk lite?
https://developers.google.com/admob/android/lite-sdk

**Comment by EliasDeuss** [comment]
@123dma admob_flutter depends on firebase_admob so it's only adding on 194kb to the size of firebase_admob .
